MSAL/src/public/MSALSilentTokenParameters.h

```diff
@@ -42,6 +42,11 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic) BOOL forceRefresh;
 
+/**
+ Ignore the redirect URI when sending requests for Native Auth flows.
+ */
+@property (nonatomic) BOOL skipSendRedirectUri;
```

Comment:
I don't think we should add this new parameter in the public silent token parameter class. External developer should not even know that a redirect URI is sent or used during the silent token retrieval.  Is there a way to avoid this change?

Reply:
Changed this implementation to not modify publicly exposed classes @nilo-ms .